apps/docs-api/typedoc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://typedoc.org/schema.json",
+  "entryPointStrategy": "packages",
+  "entryPoints": [
+    "../../packages/abi-coder",
+    "../../packages/abi-typegen",
+    "../../packages/address",
+    "../../packages/interfaces",
+    "../../packages/predicate",
+    "../../packages/account",
+    "../../packages/program",
+    "../../packages/contract",
+    "../../packages/script",
+    "../../packages/utils",
+    "../../packages/crypto",
+    "../../packages/errors",
+    "../../packages/hasher",
+    "../../packages/math",
+    "../../packages/transactions"
```

Comment:
Currently, we only use the following packages (in: [`apps/docs/typedoc.json`](https://github.com/FuelLabs/fuels-ts/blob/db/chore/isolate-api-docs/apps/docs/typedoc.json))

 - `@fuel-ts/address`
 - `@fuel-ts/interfaces`
 - `@fuel-ts/predicate`
 - `@fuel-ts/account`
 - `@fuel-ts/program`
 - `@fuel-ts/contract`
 - `@fuel-ts/script`
 - `@fuel-ts/utils`
    
Do we want to add all our packages to the api-docs?

Reply:
I think this was intentional from Dan, but I am not aware of his motivation behind it

Reply:
@danielbate what was the motivation behind this?

Reply:
This was intentional. Our reasoning for not doing all packages last time was because it added noise to the search and sidebar. Now it's isolated, I don't see any reason we can't just expose everything to give a more complete API doc.

Reply:
Thanks for confirmation.